frontend/src/i18n.js

```diff
@@ -25,13 +25,28 @@ export const resources = {
             empty: 'Empty',
             submit: 'Submit',
             zeroPlayersNotice: "No player has cemented their legacy in to the leaderboard as the stock king. Will you be the first one?",
-            errorOccurredInServices: "An error occurred in our services"
+            errorOccurredInServices: "An error occurred in our services",
+            loading: "Loading...",
+            error: "Error",
+            "All categories": "All categories",
+            "Real Estate": "Real Estate",
+            Healthcare: "Healthcare",
+            "Basic Materials": "Basic Materials",
+            Energy: "Energy",
+            Industrials: "Industrials",
+            "Consumer Cyclical": "Consumer cyclical",
```

Comment:
You should change the new translation keys to follow the existing camel case naming convention. (eg. "Consumer Cyclical -> consumerCyclical)